website/scripts/prepare.js

```diff
@@ -0,0 +1,9 @@
+const execSync = require("child_process").execSync;
+
+function exec(cmd) {
+  execSync(cmd, { stdio: "inherit", env: process.env });
+}
+
+if (process.env.CI && process.env.TRAVIS_BRANCH === "website") {
+  exec("npm run build");
```

Comment:
For the website's `prepare` script, the package is only built when preparing from CI on the `website` branch. I believe that this is correct, but I can't actually verify without pushing to that branch.